server/lib/notifications.js

```diff
@@ -337,7 +337,9 @@ async function notifyByEmail(activity) {
         viewLatestExpenses: `${config.host.website}/${activity.data.collective.slug}/expenses#expense${activity.data.expense.id}`,
       };
       activity.data.expense.payoutMethodLabel = models.PayoutMethod.getLabel(activity.data.payoutMethod);
-      notifyUserId(activity.data.expense.UserId, activity);
+      notifyUserId(activity.data.expense.UserId, activity, {
+        replyTo: `no-reply@${activity.data.collective.slug}.opencollective.com`,
```

Comment:
@bolariin : Suggest simplifying this `replyTo` address as `no-reply@opencollective.com` as I don't see any value in using a slug as it's a no-reply address anyways. 

```suggestion
        replyTo: `no-reply@opencollective.com`,
```

Reply:
We use both versions in the code, but I tend to have a preference for `no-reply@opencollective.com` too